proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/resources/HubResponseTranslatorResource.java

```diff
@@ -52,7 +52,11 @@ public Response hubResponse(@Valid HubResponseTranslatorRequest hubResponseTrans
                 hubResponseTranslatorRequest.getRequestId(),
                 hubResponseTranslatorRequest.getDestinationUrl());
 
-        HubResponseContainer hubResponseContainer = new HubResponseContainer(hubResponseTranslatorRequest, translatedHubResponse);
+        HubResponseContainer hubResponseContainer = new HubResponseContainer(
+                hubResponseTranslatorRequest,
+                translatedHubResponse,
+                hubResponseTranslatorRequest.getDestinationUrl()
```

Comment:
We are getting the destinationUrl here via the `ESP -> gateway -> redis -> gateway -> translator`.
I'm hoping we could just use the metatron to evaluate the destination url here instead, and simplify our DTOs even more. This evaluation would be exactly as done in `EidasSamlResource.post`

Reply:
We want to respect the destinationUrl sent in the AuthnRequest, so some version of the data passing above still needs to happen. So please ignore this review.

Reply:
Yep getting this from `hubResponseTranslatorRequest` is correct as it has been selected by the ESP taking into account also the authn request - it would be nice to refactor this but for now this is the right thing to do